cirq-core/cirq/testing/consistent_qasm.py

```diff
@@ -115,10 +122,16 @@ def assert_qiskit_parsed_qasm_consistent_with_unitary(qasm, unitary):  # pragma:
         return
 
     num_qubits = int(np.log2(len(unitary)))
-    result = qiskit.execute(
-        qiskit.QuantumCircuit.from_qasm_str(qasm),
-        backend=qiskit.Aer.get_backend('unitary_simulator'),
-    )
+    qiskit_version = qiskit.version.get_version_info()
+    if qiskit_version.startswith('1.'):
+        qc = qiskit.QuantumCircuit.from_qasm_str(qasm)
+        qc.remove_final_measurements()  # no measurements allowed
+        result = qiskit.quantum_info.Operator(qc).data
```

Comment:
typo - 
```suggestion
        qiskit_unitary = qiskit.quantum_info.Operator(qc).data
```

Reply:
Fixed (along with the below line, which also needs to be adjusted)